pxemgr/ipxe_handlers.go

```diff
@@ -36,7 +36,12 @@ func (mgr *pxeManagerT) ipxeBootScript(w http.ResponseWriter, r *http.Request) {
 
 	buffer := bytes.NewBufferString("")
 	buffer.WriteString("#!ipxe\n")
-	buffer.WriteString(fmt.Sprintf("kernel %s/images/vmlinuz coreos.autologin maybe-install-coreos=stable console=ttyS0,115200n8 mayu=%s next-script=%s\n", mgr.pxeURL(), mgr.pxeURL(), mgr.pxeURL()+"/first-stage-script/__SERIAL__"))
+	if mgr.config.Network.UEFI {
+		buffer.WriteString(fmt.Sprintf("kernel %s/images/vmlinuz coreos.autologin maybe-install-coreos=stable console=ttyS0,115200n8 mayu=%s next-script=%s\n initrd=initrd.cpio.gz", mgr.pxeURL(), mgr.pxeURL(), mgr.pxeURL()+"/first-stage-script/__SERIAL__"))
```

Comment:
looking thru [CoreOS official docs](https://github.com/coreos/docs/blob/master/os/booting-with-ipxe.md) mentioned there. It looks like they use initrd everywhere by default. Despite the fact that it was added for EFI specifically, it doesn't break legacy boot.

Do we really need additional UEFI optionin Mayu? Can we just add initrd by default?